conf/hydra/help/btk_help.yaml

```diff
@@ -0,0 +1,96 @@
+# @package _group_
+
+# App name, override to match the name your app is known by
+app_name: BTK
+
+# Help header, customize to describe your app to your users
+header: == ${hydra.help.app_name} CLI ==
+
+footer: |-
+  Powered by Hydra (https://hydra.cc)
+  Use --hydra-help to view Hydra specific help
+# Basic Hydra flags:
+#   $FLAGS_HELP
+#
+# Config groups, choose one of:
+#   $APP_CONFIG_GROUPS: All config groups that does not start with hydra/.
+#   $HYDRA_CONFIG_GROUPS: All the Hydra config groups (starts with hydra/)
+#
+# Configuration generated with overrides:
+#   $CONFIG : Generated config
+#
+template: |-
+  ${hydra.help.header}
+  Welcome to the Command-Line Interface (CLI) for ${hydra.help.app_name}!
+  The CLI is designed to run BTK from start to end, starting by creating a
+  DrawBlendGenerator, followed by a MeasureGenerator, and finally calculating
+  and saving the corresponding metrics.
+
+  Assuming that BTK has been pip installed, you can run btk from the command line like e.g.
+
+  btk sampling=default draw_blends=catsim max_number=3 save_path=/directory/to/save/results cpus=1
+    verbose=False surveys=[Rubin, HST] sampling=random catalog.name=catsim
+    use_metrics=['detection', 'segmentation'] (...)
+
```

Comment:
We should also be able to write a config file and run it from there, right ? I may be useful to add a line on that.

Reply:
Yes, kinda of - it's a little difficult for the user to provide their own config from scratch actually. I was hoping to add all the required flexibility and useful defaults (in upcoming PRs) to the CLI directly so that creating a config file from scratch is unnecessary for the user.  

We could also provide a bash file that has all the defaults and that users can easily modify to their needs, but maybe let's discuss this more in the upcoming meeting and merge this for now if that's ok with you